src/viking.js

```diff
@@ -1,11 +1,63 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor (health, strength){
+                this.health = health
+                this.strength = strength
+            }
 
+        attack(){
+            return this.strength
+        }
+
+        receiveDamage(damage){
+             this.health - damage
+        }
+}
+
+    soldier1 = new Soldier(100, 100)
+    soldier1.receiveDamage(50)
+    console.log(soldier1)
+    
+    
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name, health, strength){
+    super(health, strength)
+    this.name = name
+    this.health = health
+    this.strength = strength
+
+    }
+    receiveDamage(damage){
+        this.health - damage
```

Comment:
Here, you need to reference the `this.health`, being it's value minus the damage, so, you need to store that value:
```
receiveDamage(damage) {
     this.health -= damage
     if (this.health > 0) {
     (...) }
```